packages/valory/connections/abci/tests/test_fuzz/test_fuzz.py

```diff
@@ -30,19 +28,17 @@
 )
 
 
-@pytest.mark.skip(reason="broken & takes too long time to complete on CI")
-class GrpcFuzzyTests(BaseFuzzyTests, TestCase):
+class TestFuzzyGrpc(BaseFuzzyTests):
     """Test the connection when gRPC is used"""
 
     CHANNEL_TYPE = GrpcChannel
     USE_GRPC = True
-    AGENT_TIMEOUT = 30  # 3 seconds
+    AGENT_TIMEOUT_SECONDS = 30
```

Comment:
the comment does not match the code. I changed the name of the constant and removed the comment.
https://github.com/valory-xyz/open-autonomy/blob/03dfdf3b12a1d5ace116d92ed8f26f68244429f1/packages/valory/connections/abci/tests/test_fuzz/base.py#L92-L95
then used in open-aea [is_running](https://github.com/valory-xyz/open-aea/blob/bbd8fc00e1c5ffe5c1147c2d3e342a3ecf7645cf/aea/test_tools/test_cases.py#L922-L937) and [missing_from_output](https://github.com/valory-xyz/open-aea/blob/bbd8fc00e1c5ffe5c1147c2d3e342a3ecf7645cf/aea/test_tools/test_cases.py#L876-L920)